proto/v1/anomaly_service.proto

```diff
@@ -0,0 +1,190 @@
+syntax = "proto3";
+
+package bytebase.v1;
+
+import "google/api/annotations.proto";
+import "google/api/client.proto";
+import "google/api/http.proto";
+import "google/api/field_behavior.proto";
+import "google/protobuf/field_mask.proto";
+import "google/protobuf/timestamp.proto";
+
+import "v1/common.proto";
+
+option go_package = "generated-go/v1";
+
+service AnomalyService {
+  rpc ListAnomalies(ListAnomaliesRequest) returns (ListAnomaliesResponse) {
+    option (google.api.http) = {
+      get: "/v1/{parent=environments/*/instance/*}/anomalies"
```

Comment:
Consider making this parent-less. The associated resource is associated with the anomaly message itself. When needed, we can do a search by container ID.

Reply:
Using SearchAnomalies instead. The path is `post: "/v1/anomalies/search"`